### ISSUE TITLE
LOS 16.0 (20190515) santoni [report]

### ISSUE THREAD

**Issue by t0ssox** [nichcream/rel#11]
## Report of: lineage-16.0-20190515-UNOFFICIAL-santoni

# My Config

> Recovery: OrangeFox-R9.0
> Used Root: Magisk 19.1
> Used Gapps: open_gapps-arm64-9.0-nano-20190516
> Used Firmware: fw_santoni_miui_HM4XGlobal_9.5.14_cdce816969_7.1
> Flashed in order: Firmware->Rom->Gapps->Magisk
> 
>  Fingerprint
>  [*] Goodix [ ] FPC
> 
>  Used filesystems
>  [*] ext4  [ ] f2fs
>  
>  RAM Variant
>  [*] 2GB  [ ] 3GB  [ ] 4GB

# What Works

> [Yes] Boot
> [Yes] WiFi
> [Yes] WiFi-BT same time
> [Yes] Fingerprint
> [Yes] Audio
> [Yes] USB OTG
> [Yes] USB MTP
> [Yes] SD Card
> [Yes] Deep Sleep
> [Yes] Offline Charging
> [Yes] Offline Alarm
> [Yes] SafetyNet
> [Yes] Internal error fixed
> 
> Sensors
> [Yes] Compass
> [Yes] Accelerometer
> [Yes] Vibrations
> [Yes] GPS
> [Yes] IR Blaster
> 
> Camera
> [Yes] Photos
> [Yes] Video recording
> [Yes] GCam (scrubber_GCamera2PXL)
> [Yes] GCam video recording
> [Yes] GCam HEVC (h265) video recording
> 
> RIL
> [Yes] Phone
> [Yes] Dual SIM
> [?] VoLTE/VoWiFi
> [Yes] Data
> [Yes] Sim Data Reset fixed
> 
> Bluetooth      
> [Yes] Pairing
> [Yes] Audio
> [Yes] Calling
> 
> Tethering
> [Yes] WiFi
> [Yes] USB
> 
> Playback
> [Yes] YouTube 60FPS
> [Yes] HEVC (h265)
> [Yes] AVC (h264)
> 
> Security
> **[No] Encryption**
> SEPolicies: Enforcing

# Additional

> [Yes] Setup Wizard without problems
> [Yes] Google Assistant on headphones when screen is off
> [Yes] FM Radio
> [Yes] Button remapper
> [Yes] Other customization, lineage options etc

Notes 
```
1. For first time FM Radio not worked for me (no any sound)
   but during enabling adb logcat again, started to work. 
   (probably some accident :-))
2. I'm thinking why backlight option exist in setting because
   santoni don't have backlight for hw buttons. Should be nuked from rom.
3. Encryption not working. Android logo appears then systemui restarts.
```

## Summary

> Android: 9.0.0
> Ready for daily use: Of course :3

**Thanks for hard work to port 4.9 linux kernel to santoni. You must be very talented in knowing linux stuff to get this work. I hope you will be official LineageOS maintainer for santoni and/or land. I'm attaching some logcats if you want to check if everything is correct in system. Thanks for reading! 🙂**

![Screenshot_20190516-181544_Settings](https://user-images.githubusercontent.com/17672296/57876157-f238d280-7814-11e9-97fb-8d4f5b7db91b.png)

More screenshots (Including kernel adiutor ss): https://imgur.com/a/uj1FUPA

[logcat_fmradio_with_b_all.txt](https://github.com/nichcream/rel/files/3188350/logcat_fmradio_with_b_all.txt)
[logcat_fmradio.txt](https://github.com/nichcream/rel/files/3188351/logcat_fmradio.txt)
[logcat_encryption.txt](https://github.com/nichcream/rel/files/3188352/logcat_encryption.txt)
[logcat.txt](https://github.com/nichcream/rel/files/3188353/logcat.txt)
[logcat_boot.txt](https://github.com/nichcream/rel/files/3188354/logcat_boot.txt)

**Comment by ghost** [comment]
Did your goodix fp works? Mine didn't work, even tried with flashing your Fingerprint Reseter. Some users can't boot it with f2fs. And when i rebooted, the battery stats are reseted, this issue found after charging my phone. That's it

**Comment by t0ssox** [comment]
@AKD264 are you using any kind of mods like viper ? They can cause some issues. And give logcats to goodix.

**Comment by ghost** [comment]
@t0ssox yeah, im using magisk module (qcmod and viper4android). I'll be try to use ur firmware and give logs
